dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -60,13 +61,17 @@ public static void main(String[] args) {
 
     @EventListener
     public void run(ApplicationReadyEvent readyEvent) {
-        log.info("Alert server is staring ...");
         alertPluginManager.start();
-        alertRegistryClient.start();
-        alertBootstrapService.start();
-        listenerEventPostService.start();
-        alertRpcServer.start();
-        log.info("Alert server is started ...");
+
+        if (this.getClass().getName()
+                .contains(((SpringApplication) readyEvent.getSource()).getMainApplicationClass().getName())) {
+            log.info("Alert server is staring ...");
+            alertRegistryClient.start();
+            alertBootstrapService.start();
+            listenerEventPostService.start();
+            alertRpcServer.start();
+            log.info("Alert server is started ...");
+        }
```

Comment:
Why do this change?

Reply:
Pre-design is to import the alert server in API-server, which will run the alert server auto, I just need the alert plugin manager and not need another component, so add this code. But change to use rpc does not need this change.